source/meta.tex

```diff
@@ -1712,22 +1712,32 @@
 \indexlibraryglobal{add_const}%
 \tcode{template<class T>\br
  struct add_const;}                 &
- If \tcode{T} is a reference, function, or top-level const-qualified
- type, then \tcode{type} denotes \tcode{T}, otherwise
- \tcode{T const}.                                                           \\  \rowsep
+ The member typedef \tcode{type} denotes \tcode{const T}.
+\begin{tailnote}
+\keyword{const} has no effect when \tcode{T} is a reference, function, or
+top-level const-qualified type.
+\end{tailnote}
```

Comment:
I'm not sure whether it's still worthy to talk about reference, function, or top-level _cv_-qualified types.

Also, I feel that it might make sense to group `type_identity` with these traits.